config/environments/production.rb

```diff
@@ -122,7 +122,7 @@
   # Skip DNS rebinding protection for the default health check endpoint.
   # config.host_authorization = { exclude: ->(request) { request.path == "/up" } }
 
-  config.active_storage.service = :production
+  config.active_storage.service = :local
```

Comment:
by reading the Rails's `storage.yml` example, the `local` can be seen as local disk, not local env, so i guess it also makes sense that production env uses local storage.

https://github.com/rails/rails/blob/97dac99b5e80cc0aab3764e86fbcd12f4c316112/railties/lib/rails/generators/rails/app/templates/config/storage.yml.tt#L5-L15